samples/bluetooth/mesh_demo/src/microbit.c

```diff
@@ -29,9 +31,13 @@
 #define SEQ_PAGE     (NRF_FICR->CODEPAGESIZE * (NRF_FICR->CODESIZE - 1))
 #define SEQ_MAX      (NRF_FICR->CODEPAGESIZE * 8 * SEQ_PER_BIT)
 
-static const struct device *gpio;
-static const struct device *nvm;
-static const struct device *pwm;
+static const struct gpio_dt_spec button_a =
+	GPIO_DT_SPEC_GET(DT_NODELABEL(buttona), gpios);
+static const struct gpio_dt_spec button_b =
+	GPIO_DT_SPEC_GET(DT_NODELABEL(buttonb), gpios);
+static const struct device *nvm =
+	DEVICE_DT_GET(DT_CHOSEN(zephyr_flash_controller));
```

Comment:
Was thinking we need to replace DT_CHOSEN_ZEPHYR_FLASH_CONTROLLER_LABEL with `DT_CHOSEN_ZEPHYR_FLASH_CONTROLLER` or `DT_CHOSEN_ZEPHYR_FLASH_CONTROLLER_NODE` in `include/devicetree/zephyr.h`

Reply:
> Was thinking we need to replace DT_CHOSEN_ZEPHYR_FLASH_CONTROLLER_LABEL

Agree, but out of scope for this PR.